src/main/webapp/index.jsp

```diff
@@ -42,11 +43,19 @@
         Through our site, you will be able to connect with the world through online chat!
       </p>
       <ul>
-        <li><a href="/login">Login</a> to get started.</li>
-        <li>Go to the <a href="/conversations">conversations</a> page to
+        <% if(request.getSession().getAttribute("user") != null){ %>
+          <li>Go to the <a href="/conversations">conversations</a> page to
             create or join a conversation.</li>
-        <li>View the <a href="/about.jsp">about</a> page to learn more about the
+          <li>View the <a href="/about.jsp">about</a> page to learn more about the
             project.</li>
+          <li>View the <a href="/profile.jsp">My profile</a> page to view your profile.</li>
```

Comment:
Just a small comment: You use the word "view" twice.  Maybe "... to see your profile." ?

Reply:
Had that there in the testing phase to see how everything worked. Just changed it. Thanks!